### PR TITLE
[teleop] Updated GUI README

### PR DESCRIPTION
**Pull request by ErikLytle**
Added in missing LCMs for the Science GUI, updated subscriptions and publishers

### CODE REVIEW

File: base_station/gui/README.md [hunk 1]
```diff
@@ -3,13 +3,34 @@
 ### Base Station GUI LCMs
 
 #### All GUIs:
-Camera 
+ArmControlState
+"/arm_control_state"
+File: ArmControlState.lcm
+Subscriptions: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:9]: jetson/teleop, jetson/ra_kinematics

File: base_station/gui/README.md [hunk 2]
```diff
@@ -3,13 +3,34 @@
...
+Publishers: ArmControls.vue
+Usage: Selecting open/closed loop
+
+ArmControlState
+"/arm_control_state_to_gui"
+File: ArmControlState.lcm
+Subscriptions: ArmControls.vue, SAArm.vue,
+Publishers: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:17]: jetson/ra_kinematics

File: base_station/gui/README.md [hunk 3]
```diff
@@ -3,13 +3,34 @@
...
 "/camera_servos"
 File: CameraServos.lcm
 Subscritions: SATask.vue, AutonTask.vue, RATask.vue
 Publishers: N/A
 Usage: 
 
+Cameras
+"/cameras_cmd"
+File: Cameras.lcm
+Subscriptions: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:30]: jetson/cameras? Ask Guthrie which program listens to this

File: base_station/gui/README.md [hunk 4]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
+Publishers: ArmControls.vue
+Usage: Values based on xbox controller input
+
 #### Science GUI:
+CarouselData
+"/carousel_data"
+File: CarouselData.lcm
+Subscriptions: Carousel.vue
+Publishers: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:116]: Science Bridge

File: base_station/gui/README.md [hunk 5]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
+File: CarouselOpenLoopCmd.lcm
+Subscriptions: N/A
+Publishers: Carousel.vue
+Usage: Sends out throttle for Carousel control
+
+FuseData
+"/fuse_data"
+File: FuseData.lcm
+Subscriptions: PDBFuse.vue
+Publishers: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:137]: Can leave as N/A for now

File: base_station/gui/README.md [hunk 6]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
 Subscriptions: Chrlophyll.vue
-Publishers: Chrlophyll.vue, Amino.vue, OdometryReadingSA.vue
+Publishers: Chrlophyll.vue, Amino.vue, OdometryReadingSA.vue, Raman.vue, ScoopUV.vue
 Usage: Enabling and disabling Raman test, UV lights, Thermistor Heaters
 
+PDBData
+"/pdb_data"
+File: PDBData.lcm
+Subscriptions: PDBFuse.vue
+Publishers: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:179]: Can leave as N/A for now

File: base_station/gui/README.md [hunk 7]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
+Subscriptions: N/A
+Publishers: Amino.vue
+Usage: Sends bool with newly selected heater autoshutdown on/off value
+
 MosfetCmd
 "/mosfet_cmd"
 File: MosfetCmd.lcm
 Subscriptions: Chrlophyll.vue
-Publishers: Chrlophyll.vue, Amino.vue, OdometryReadingSA.vue
+Publishers: Chrlophyll.vue, Amino.vue, OdometryReadingSA.vue, Raman.vue, ScoopUV.vue
```
Comment (CameronTressler) [inline base_station/gui/README.md:172]: I don't think chlorophyll subscribes to mosfet_cmd? Or if it does we should maybe stop doing that. Also watch the spelling on chlorophyll, I know this was already here.

File: base_station/gui/README.md [hunk 8]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
+File: PDBData.lcm
+Subscriptions: PDBFuse.vue
+Publishers: N/A
+Usage: Values are displayed in PDBFuse table
+
+SAPosition
+"/sa_position"
+File: SAPosition.lcm
+Subscriptions: SAArm.vue
+Publishers: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:186]: jetson/nucleo_bridge

File: base_station/gui/README.md [hunk 9]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
+Usage: Sends the current positions of each joint in radians
+
+ServoCmd
+"/servo_cmd"
+File: ServoCmd.lcm
+Subscriptions: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:192]: jetson/science_bridge

File: base_station/gui/README.md [hunk 10]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
 ThermistorData.lcm
-Subscriptions: Amino.lcm
+Subscriptions: Amino.vue, GenerateReport.vue
 Publishers: N/A
 Usage: Used to read in values for thermistor temperatures
 
+Xbox
+"/sa_control"
+File: Xbox.lcm
+Subscriptions: N/A
```
Comment (CameronTressler) [inline base_station/gui/README.md:220]: jetson/teleop

File: base_station/gui/README.md [hunk 11]
```diff
@@ -53,41 +88,139 @@ Publishers: N/A
...
+Subscriptions: SATask.vue
+Publishers: N/A
+Usage: Data from spectral triad end effector displayed in SpectralData vue component
 
 ThermistorData
-"thermistor_data"
+"/thermistor_data"
 ThermistorData.lcm
-Subscriptions: Amino.lcm
+Subscriptions: Amino.vue, GenerateReport.vue
```
Comment (CameronTressler) [inline base_station/gui/README.md:213]: Most of these science ones that affect things in the ISH box should be communicating with jetson/science_bridge

**Review by CameronTressler — CHANGES_REQUESTED** [review]

File: base_station/gui/README.md (hunk 3 above)
Comment (tabiosg) [reply base_station/gui/README.md:30]: yes jetson/cameras

File: base_station/gui/README.md (hunk 5 above)
Comment (tabiosg) [reply base_station/gui/README.md:137]: jetson/pdb_fuse_bridge or N/A

**Pull request closed without merging**